checks/pullPolicyNotAlways.yaml

```diff
@@ -9,3 +9,7 @@ schema:
   properties:
     imagePullPolicy:
       const: Always
+mutations:
+  - op: add
+    path: /spec/containers/0/imagePullPolicy
```

Comment:
```suggestion
    path: /imagePullPolicy
```

Reply:
Per offline discussion, let's:

* scope this JSON patch down to the container level to match `target` in the check
* return this JSON Patch inside the `ResultMessage` struct
* once all schemas have been applied and you get back a `Result` struct for each resource, which includes each of the applicable JSON Patches:
  * iterate through the `Result` and apply the patches that should be applied based on the user's configuration
  * you'll need to change the JSON Patch `path` field if it's a `PodResult` or `ContainerResult`
    * e.g. `path` for PodResult is different for CronJobs vs Deployments
    * and `path` for ContainerResult will need to use `ContainerResult.Name` to find the right index in the `containers` array